src/cocoa/toga_cocoa/dialogs.py

```diff
@@ -132,4 +132,19 @@ def open_file(window, title, file_types, multiselect):
         filename_or_filenames = (paths if multiselect else
                                  panel.URL.path)
         return filename_or_filenames
+
+
+def select_folder(window, title):
+    dialog = NSOpenPanel.alloc().init()
+    dialog.title = title
+
+    dialog.canChooseFiles = False
+    dialog.canChooseDirectories = True
+    dialog.resolvesAliases = True
+    dialog.allowsMultipleSelection = True
+
+    result = dialog.runModal()
+
+    if result == NSFileHandlingPanelOKButton:
```

Comment:
At column 18: (F405) 'NSFileHandlingPanelOKButton' may be undefined, or defined from star imports: .libs